pkg/cluster/internal/kubeadm/config.go

```diff
@@ -74,11 +76,14 @@ clusterName: "{{.ClusterName}}"
 bootstrapTokens:
 - token: "{{ .Token }}"
 {{ if .ControlPlaneEndpoint -}}
-controlPlaneEndpoint: {{ .ControlPlaneEndpoint }}
+controlPlaneEndpoint: "{{ .ControlPlaneEndpoint }}"
```

Comment:
was the quoting needed here?

Reply:
I had to quote the ControlPlaneEndpoint because it failed to create the JSON from the template using IPv6 addresses

Reply:
ok, understood.